api/routes.ts

```diff
@@ -2,13 +2,16 @@ import * as express from 'express'
 import * as amendedJurisdictions from './amendedJurisdictions'
 import authInterceptor from './lib/middleware/auth'
 import * as proxy from './lib/proxy'
+import * as searchCases from './searchCases'
 
 const router = express.Router({ mergeParams: true })
 
 router.use(authInterceptor)
 
 router.get('/caseworkers/:uid/jurisdictions', amendedJurisdictions.getJurisdictions)
 
+router.post('/internal/searchCases', searchCases.getCases)
```

Comment:
@andywilkinshmcts is this what you had in mind? We're intercepting the proxy to handle elastic search query on the node layer